Marlin/src/module/delta.cpp

```diff
@@ -242,10 +242,12 @@ void home_delta() {
   #endif
 
   // Move all carriages together linearly until an endstop is hit.
+  TERN_(HAS_HOMING_CURRENT, set_homing_current(Z_AXIS)); // This sets homing current to all motors
```

Comment:
"all motors"? … or _all Z motors_ ?

Reply:
This is the Delta module. On such a printer, if I'm correct, moving along the Z axis requires all A, B and C motors at the same time. So in `motion.cpp` it is set that setting current on the Z axis would set it to all these 3 motors

```
#elif ANY(AXEL_TPARA, DELTA)
    // Special handling of TPARA kinematics
    // Z_AXIS is a special mode to set homing current to all axis
      if (axis == A_AXIS || axis == Z_AXIS) {
        #if HAS_CURRENT_HOME(X)
          _SAVE_SET_CURRENT(X);
        #endif
      }
      if (axis == B_AXIS || axis == Z_AXIS) {
        #if HAS_CURRENT_HOME(Y)
          _SAVE_SET_CURRENT(Y);
        #endif
      }
      if (axis == C_AXIS || axis == Z_AXIS) {
        #if HAS_CURRENT_HOME(Z)
          _SAVE_SET_CURRENT(Z);
        #endif
      }
```

The TPARA kinematic is set to the same stuff from what I could read in `scara.cpp`, I tried to find material about this printer but it was quite hard ^_^ By default, I chose to use the same "trick", but that could be incorrect for that printer type. Any input on the matter is more than welcome :D